v22.1/show-schedules.md

```diff
@@ -130,6 +130,6 @@ To view a specific schedule, use the schedule's `id`:
 - [`RESUME JOB`](pause-job.html)
 - [`CANCEL JOB`](cancel-job.html)
 - [Take Full and Incremental Backups](take-full-and-incremental-backups.html)
-- [Row-Level TTL](row-level-ttl.html) <span class="version-tag">New in v22.1</span>
+- **New in v22.1:** [Row-Level TTL](row-level-ttl.html)
```

Comment:
I don't think we should change from using our standard "new in XX" version tag just so there can be a colon.  For the following reasons:

1. it's different visually from the colored tag we normally use, and thus doesn't send the same signal to the reader
2. less important but still a thing: the existing tag is used by our ["bump docs version" script](https://github.com/cockroachdb/ed-tools/blob/main/docs/bump-major-version/bump_major_version.py#L181)

I'm fine with adding a colon to all uses of the existing tag though

Reply:
I'm just implementing what's in the style guide: https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#version-tags 

Note: Do not use version tags in bulleted lists items. To denote a new feature in a bulleted list, start the bulleted item with "**New in vX.X:**".